### PR TITLE
UCP/UD: disabe multi-lane for UD TLS

### PR DESCRIPTION
**Pull request by hoopoepg**
- due to issues in US progress multilane is disabled for
  this transport

fixes https://github.com/openucx/ucx/issues/2229

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6056/ for details (Mellanox internal link).

File: src/ucp/wireup/select.c [hunk 1]
```diff
@@ -800,6 +800,12 @@ static double ucp_wireup_am_bw_score_func(ucp_context_h context,
     return size / time * 1e-5;
 }
 
+static int ucp_wireup_is_ud_lane(ucp_context_h context, ucp_rsc_index_t rsc_index)
+{
+    return !strstr(context->tl_rscs[rsc_index].tl_rsc.tl_name, "ud") ||
+           !strstr(context->tl_rscs[rsc_index].tl_rsc.tl_name, "ud_mlx5");
```
Comment (bureddy) [inline src/ucp/wireup/select.c]: is this logic correct?  is this func need to return for 1  ud/ud_mlx5 ?  it that is that case, it seems it wrong logic

Comment (bureddy) [reply src/ucp/wireup/select.c]: I think it should be 's/strstr/strcmp'

Comment (hoopoepg) [reply src/ucp/wireup/select.c]: my fault :(
fix is on the way: https://github.com/openucx/ucx/pull/2235

Comment (bureddy) [reply src/ucp/wireup/select.c]: Thanks. I also added same fix in #2214  to fix the PR Jenkins

**Pull request merged**